src/components/nfts/NFTCard.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react'
+import { NFT } from '../../reducers/nftReducer'
+import ArrowForward from '@material-ui/icons/ArrowForward'
+import NFTAttribute from './NFTAttibute'
+
+import NoImageFound from '../../assets/no-image-found.png'
+
+import './NFTCard.scss'
+import TextBreakable from '../text-breakable/TextBreakable'
+
+type Props = {
+  data: NFT
+}
+
+const NFTCard: React.FC<Props> = ({ data }) => {
+  function handleOnError({
+    currentTarget,
+  }: React.SyntheticEvent<HTMLImageElement, Event>) {
+    currentTarget.onerror = null
+    currentTarget.src = NoImageFound
+  }
+
+  return (
+    <div className="card-container">
+      <div className="card-image-container">
+        <div className="token-image">
+          <img src={data.image} alt={data.name} onError={handleOnError} />
+        </div>
+      </div>
+      <div className="card-header">
+        <div className="card-header-content">
+          <div className="collection-image-container">
+            {/* <div className="collection-image">
```

Comment:
Please don't commit commented code without an a "TODO" message explaining when it will be uncommented